### ISSUE TITLE
Tab order

### ISSUE THREAD

**Issue by getflourish** [sebastiangreger/kirby3-commentions#98]
When tabbing from one field to the next: "name" → "email" → "comment", there are two tabs necessary to get to the "comment" field. This is because the `honeypot` field is placed before the `comment` field. 

I can imagine how this prevents spam, but it also can lead to mistakes by tabbing to the invisible honeypot and typing something. The form will then not be submitted.

What’s your recommendation to change the order?

**Comment by getflourish** [comment]
It’s essentially about the order of the honeypot field in: 

`commentions-form.php`

```
<div class="commentions-form-honeypot">
        <label for="website"><?= t('commentions.snippet.form.honeypot') ?></label>
        <input type="url" id="website" name="website">
      </div>
```

**Comment by sebastiangreger** [comment]
Thank you for pointing this out. While screen reader users would understand what to do (as the label tells to leave that field empty), this is indeed a UX issue for sighted keyboard users.

I believe that adding `tabindex="-1"` to the honeypot field solves this without causing accessibility issues. Remains to be seen whether some smart spambots would also skip such field, in which case this issue would have to be revisited with a more comprehensive solution.

This update is included in the latest [v1.0.4 release](https://github.com/sebastiangreger/kirby3-commentions/releases/tag/v1.0.4); please re-open if you see a need for further changes.

**Comment by getflourish** [comment]
Thank you! 

I don’t know how bots operate, but assume they rather jump from one field to the other using JavaScript. This would be way safer for bots who would otherwise fail to complete "inaccessible" forms. On the other hand, I can also imagine smarter bots that emulate keypresses :D I think either using `tabindex="-1` should be fine.
